### DISCUSSION TITLE
How to enable Ethernet?

### DISCUSSION THREAD

**Discussion by Joern-W** [emsesp/EMS-ESP32#1370] [General]
Today, I finally installed an Ethernet cable to my boiler room. I connected the EMS-ESP32 to the Ethernet cable and rebooted the device. However, it still connects via Wi-Fi. Is there any setting I have to adjust? In the network settings of the EMS-ESP32 there is no option to enable Ethernet.

Details:
The device is a BBQKees Gateway E32 with Ethernet support (https://bbqkees-electronics.nl/product/gateway-e32-ethernet-edition/)
EMS-ESP Version: 3.6.2
Platform (Platform / SDK): ESP32 / 4.4.5.230901
System Betriebszeit: 000+00:02:48.072
CPU Frequenz: 240 MHz
freier RAM Speicher (Gesamt / max. Block): 125 KB / 61 KB
Flash Speicher (Größe / Geschwindigkeit): 4.096 KB / 40 MHz
Programm (Genutzt / Frei): 1.547 KB / 437 KB
Dateisystem (Genutzt / Frei): 24 KB / 40 KB

I checked the cable using another device, the cable is okay.

What has to be done to enable ethernet?

Regards
Jörn

**Comment by proddy** [comment]
leave the SSID blank

**Comment by Joern-W** [comment]
Great, that worked. Thank You!
My problem was that the hint is only visible when the SSID is empty. Maybe you could put an additional hint on the network settings page?

**Reply by proddy** [reply]
glad it worked. I will need to update the Wiki to explain this clearly.

The hint is also in the title

![image](https://github.com/emsesp/EMS-ESP32/assets/1230712/b6597dc6-ba77-4dca-b8f4-2e80885aa572)

**Reply by MichaelDvP** [reply]
The addition `and enable ETH` is still missing for some translations.

**Reply by proddy** [reply]
I'll add English ones for now
